packages/expo-mail-composer/src/MailComposer.types.ts

```diff
@@ -8,8 +8,8 @@ export type ComposeOptions = {
   attachments?: string[];
 };
 
-export type ComposeResult = {
-  status: ComposeStatus;
+export type MailComposerResult = {
+  status: MailComposerStatus;
 };
 
-export type ComposeStatus = 'undetermined' | 'sent' | 'saved' | 'cancelled';
+export type MailComposerStatus = 'undetermined' | 'sent' | 'saved' | 'cancelled';
```

Comment:
Hey I stopped for a while here and came up with an idea how to make `MailComposerStatus` both a `type` an a `value` at the same time with the same name 🎉 
Take a look here at this [Typescript Playground](https://www.typescriptlang.org/play/index.html#code/KYDwDg9gTgLgBAYwgOwM7wLIEMCWAbAYQgFtJVgoBlGLGAV1TgF44BvAKDjgFUA5AEQCiAFUEAlDAEleg-gC44AcjrIAJsBgViOZMFWKANJziVBvYQsXlkMQ8coBBAGqzLqLADc9drgQe8CQQAZINclBCxkBGA8PG8jAF84LEYkNBh2dlBIWDgYAE8wYDhsfCJSCHIqGnpGFgKiiAAzEtxCEjIKaloGAG0Aa2B85rzC4BHS9oqq7tqAXUymlQQYHBQ4JogIAAoAIywoBUnyzuqe1ABKNkyExa3t447KrpqGADpTcwuAbiA)